src/main/kotlin/net/lumalyte/lg/interaction/menus/guild/GuildStatisticsMenu.kt

```
@@ -247,9 +247,40 @@ class GuildStatisticsMenu(private val menuNavigator: MenuNavigator, private val
+                .lore("§7K/D Ratio: §e${decimalFormat.format(killStats.killDeathRatio)}")
+                .lore("")
+                .lore("§7Kill Efficiency: §f${calculateEfficiency(killStats)}%")
+            pane.addItem(GuiItem(summaryItem), 4, 1)
+
+            val backItem = ItemStack.of(Material.ARROW)
+                .name("§cBack to Statistics")
+                .lore("§7Return to guild statistics")
+            pane.addItem(GuiItem(backItem) { open() }, 4, 3)
+
+            gui.show(player)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)